### ISSUE TITLE
Staging Review finding: Icon on map image not actually a button, not focusable.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#87426]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Sitewide Facilities
**Product:** VBA Regional Offices
**Feature:** Site Modernization

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the "Location and contact information" section, the image of a map has a `zoom_out_map` icon overlayed on top of it, styled to look like a button (including hover state). However:
- The icon isn't actually a button, it's just an element inside the link containing the map.
- Since it's not actually a button, the icon isn't focusable for keyboard users who might expect ot be able to tab to it.
- Clicking the link doesn't zoom anything, it navigates you to a new place.

This doesn't match with user expectations (clicking does something unexpected), and might frustrate keyboard-only users who may believe there's some additional functionality that they can't get to.

## Recommended action

I would recommend harmonizing with [ongoing work for the Vet Centers template](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18268#issuecomment-2153102587), which recommends removing the icon. I don't think it adds much value here.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 2.1.1 Keyboard - Level A](https://www.w3.org/TR/WCAG21/#keyboard)
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jilladams** [comment]
Tracking in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18268
